src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+* {
+  --active-color: #00acdc;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 50px;
+}
+
+.header {
+  display: flex;
+  width: 1200px;
+  margin: 0 auto;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  text-align: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
```

Comment:
add font-family style to body or html selector. then you can use font-family: inherit
do not forget to add more fallback fonts
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/854b9ab6-a533-4df4-9ce3-7d99f375b370)